src/PAL/FileSystem/nanoPAL_FileSystem.cpp

```
@@ -237,7 +237,7 @@ bool FileSystemVolumeList::AddVolume(
     // if the name is already taken, there is no need to add the volume
     if (nameOK == false)
```

Clang (14.0.6)

[warning] redundant boolean literal supplied to boolean operator

(readability-simplify-boolean-expr)